### ISSUE TITLE
Appeding Model Path on generating ID from scoutKey with Modle

### ISSUE THREAD

**Issue by MadaniTech** [algolia/scout-extended#209]
Scout Extended is wonderful with its latest features to use Algolia Search API in laravel. But It has a bad approach to save value of Model path with ScoutKey.

On encryption it is using following in **ObjectIdEncrypter**:
` public static function encrypt($searchable, int $part = null): string
    {
        $scoutKey = method_exists($searchable, 'getScoutKey') ? $searchable->getScoutKey() : $searchable->getKey();

        $meta = [get_class($searchable->getModel()), $scoutKey];

        if ($part !== null) {
            $meta[] = $part;
        }

        return implode(self::$separator, $meta);
    }`
Because of this ObjectID is importing like this:
`objectID: {ModelPath}::ScoutKey`
on importing and searching results also show a string including Model Path.
Any body can help in this matter.

**Comment by nunomaduro** [comment]
Hey @MadaniTech, what issues are caused in your side about having the model path on the objectID?

**Comment by MadaniTech** [comment]
Thanks for the response @nunomaduro. Is this right to show the path on response while fetching the results from search query? I am writing my logics which also impact on directory path and on the fly I am showing this path as objectID in response.

**Comment by nunomaduro** [comment]
Yeah - at the moment, moving the model from directory will force you to reimport all records. Maybe we can improve this in the future.

**Comment by MadaniTech** [comment]
Can I modify it for the time for my project? In case please help me out. Thanks @nunomaduro.
